packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js

```diff
@@ -33,15 +33,18 @@ function SlideTutorial (props) {
       pad={pad}
       width={width}
     >
-      {isThereMedia &&
-        <Media
-          alt={counterpart('SlideTutorial.alt', { activeStep })}
-          fit='cover'
-          height={200}
-          src={medium.src}
-        />}
-      <Heading level='3' margin='20px 0 10px 0'>{counterpart('SlideTutorial.heading', { projectDisplayName })}</Heading>
-      <StyledMarkdownWrapper aria-live='polite' autoFocus height='100%' overflow='auto'>
+      <StyledMarkdownWrapper aria-live='polite' autoFocus height='100%' overflow={{ horizontal: 'hidden', vertical: 'auto' }}>
+        {isThereMedia &&
+          <Media
+            alt={counterpart('SlideTutorial.alt', { activeStep })}
+            fit='cover'
+            height={200}
+            src={medium.src}
+          />}
+        {activeStep === 0 &&
+          <Heading level='3' margin='20px 0 10px 0'>
```

Comment:
we can use the named margins here! `margin={{ bottom: 'xsmall', top: 'small' }}`

https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-grommet-theme/src/index.js#L110